index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Marketing Acessability website</title>
+  </head>
+
+  <!-- header section with nav bar containing links -->
+  <body>
+    <!-- nav - header tag-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- navigation bar with the website links -->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management">Online Reputation Management</a>
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </nav>
+  </header>
+    <!-- nav tag -->
+    <div class="hero"></div>
+    <div class="content">
```

Comment:
Consider using a semantic html tag here instead of the `div`. Although a `div` works fine, it doesn't tell us what kind of section we are looking at. You can see examples of semantic html here which may help you choose: https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
added aside tag to div classes, i didn't know wether to add to all as I think the div is better than section?